recipes/nmslib/all/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.4)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
```

This could help the shared problem

```suggestion

set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

---

I don't want to add this as this is clearly not considered by the library author. It will be easy to add in the future if someone needs it